src/Juvix/Compiler/Pipeline.hs

```diff
@@ -199,33 +203,109 @@ coreToGeb spec = Core.toGeb >=> return . uncurry (Geb.toResult spec) . Geb.fromC
 -- Run pipeline
 --------------------------------------------------------------------------------
 
-runIOEither :: forall a. BuiltinsState -> EntryPoint -> Sem PipelineEff a -> IO (Either JuvixError (Artifacts, a))
-runIOEither builtinsState entry =
+-- | It returns `ResolverState` so that we can retrieve the `juvix.yaml` files,
+-- which we require for `Sope` tests.
```

Comment:
typo in comment: Sope -> Scope

Reply:
fixed https://github.com/anoma/juvix/pull/1911/commits/342cc1b22c78a7bfadc7cb46c1fe24c99429d2b0